### PR TITLE
Survey

### PR DESCRIPTION
**Pull request by maxxof**
- Added survey link component to homepage
- Added table urls to database that stores different urls
- Added api for fetching the url from the backend
- Added tests for survey link

### CODE REVIEW

**Comment by Robomarti** [comment]
huhhuh ku 5 eslint errorii siel pilaamas mergee

File: src/lang/i18n.js
Comment (Robomarti) [inline src/lang/i18n.js:1]: default language must be finnish

**Review by Robomarti — COMMENTED** [review]
Docstrings for different responses are good, but I'm pretty sure that the default language must be Finnish. Maybe in the e2e tests we can use the localized texts instead of the hardcoded English texts

File: src/lang/i18n.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    .use(initReactI18next)
+    // init i18next
+    // for all options read:
+    // https://www.i18next.com/overview/configuration-options
+    .init({
+        compatibilityJSON: 'v3',
+        resources,
+        fallbackLng: 'fin',
+        lng: 'eng', // default language to use.
+        supportedLngs: ['fin', 'eng'],
```
Comment (PyryL) [inline src/lang/i18n.js:26]: These could be changed to [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) standard codes, such as `fi` and `en`, but good for now.

File: backend/routes/url.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+/**
+ * Route for requesting urls.
+ * @param {Object} req - The request object.
+ * @param {Object} req.body - The body of the request.
+ * @param {Object} res - The response object.
+ * @returns {Object} 200 - Success status.
+ * @returns {Object} 404 - Url not found.
+ */
+router.get('/api/url/:urlName', async (req, res) => {
```
Comment (PyryL) [inline backend/routes/url.js:14]: This could be renamed to something more descriptive, such as `/api/surveyUrl/:urlName`, but good for now.

**Review by PyryL — APPROVED** [review]
Quite a lot of changes but overall it's ok.

**Pull request merged**